src/daemons/MetaDaemon.cpp

```diff
@@ -69,9 +76,50 @@ std::unique_ptr<nebula::kvstore::KVStore> initKV(std::vector<nebula::HostAddr> p
                                                         localhost,
                                                         threadManager);
     if (!(kvstore->init())) {
-        LOG(ERROR) << "nebula store init failed";
+        LOG(ERROR) << "Nebula store init failed";
         return nullptr;
     }
+
+    LOG(INFO) << "Waiting for the leader elected...";
+    nebula::HostAddr leader;
+    while (true) {
+        auto ret = kvstore->partLeader(0, 0);
```

Comment:
what mean about (0, 0)?

Reply:
For meta,  we only have one space (0) and one part (0)

Reply:
maybe we can replace this magic number to some readable word, that coder understand it more easy.

Reply:
Or make the previous reply as a comment in front of this line

Reply:
> For meta, we only have one space (0) and one part (0)

Put this as a comment in the code?

Reply:
Good suggestion.  I will fix it.